12/problem12.rb

```diff
@@ -0,0 +1,14 @@
+def table (num)
+  i=1
+  while i<=10
+    temp=num*(i)
+    puts "#{num} * #{i} = #{temp}"
+    i+=1
+  end
+end 
+
+
+puts "Enter Number to write table:"
+number=gets.chomp!.to_i
+
+table(number)
```

Comment:
@Haider0007 using method to print a table of a number is a very good approach. Well done!